src/database/seed/data.ts

```diff
@@ -186,6 +186,16 @@ export const models: SeedClientOptions["models"] = {
       published: () => faker.date.past(),
     },
   },
+  election: {
+    data: {
+      createdAt: () => faker.date.recent({ days: 14 }),
+      expiresAt: () => faker.date.soon({ days: 14 }),
+      markdown: () => "- " + faker.lorem.sentences({ min: 3, max: 7 }, "\n- "),
+      markdownEn: () =>
+        "- " + faker.lorem.sentences({ min: 3, max: 7 }, "\n- "),
+      link: () => faker.internet.url(),
+    },
+  },
```

Comment:
Added elections to our database seeding to have some test data to look at in development.